### ISSUE TITLE
feat: Create a login Page

### ISSUE THREAD

**Issue by khushishikhu** [Feminine-Divine/periodo#12]
## Description 
Create a Login Page for the app in a separate folder.

## Reference
For design please visit [here](https://codepen.io/) or you can also present your idea (discuss with the Mentor)

**Comment by kunal299** [comment]
@khushishikhu Could you please assign this issue to me as a part of gssoc'21?

**Comment by khushishikhu** [comment]
@kunal299 assigning it to you. Happy Coding!!

**Comment by sahildotexe** [comment]
Hey !! I am a participant of GSSOC'21 , can you assign this issue to me?

**Comment by Esha-Sherring** [comment]
I would like to work on this issue

**Comment by drishtipeshwani** [comment]
Hey Khushi !! Could you assign this to me I would like to work on this issue

**Comment by prerna2712** [comment]
I would like to work on this issue, can we divide the work accordingly?

**Comment by khushishikhu** [comment]
Yes @prerna2712 Assigning this to you also as this is a issue needs more work

**Comment by prerna2712** [comment]
> Yes @prerna2712 Assigning this to you also as this is a issue needs more work

Is there a predefined design, or we can make anything that matches the theme?

**Comment by khushishikhu** [comment]
@prerna2712 you can make anything it's up to your choice. Then we will look into it

**Comment by ambivert-sam** [comment]
Hey !! I am a participant of GSSOC'21 ,  I would like to work on this issue?

**Comment by khushishikhu** [comment]
@ambivert-sam assigning this to you

**Comment by ambivert-sam** [comment]
Mam please review it
![Screenshot (157)](https://user-images.githubusercontent.com/74984661/111902452-8ef8d680-8a63-11eb-9c48-ef60d7063ebe.png)

**Comment by ambivert-sam** [comment]
@khushishikhu mam can I make PR?

**Comment by khushishikhu** [comment]
@ambivert-sam Yes please go ahead. I would be happy to show that in action with this project

**Comment by khushishikhu** [comment]
@ambivert-sam  I have checked your PR to which somehow I feel like you didn't know that here in this Project we do not use html file to create the login page as this project has built with the Next.js framework. So if possible please make the PR of try to implement your ideas with Next.js. To know further about Next.js you can read it's documentation, it is easily available on the internet. I hope you got what I am trying to say.

**Comment by ambivert-sam** [comment]
> @ambivert-sam I have checked your PR to which somehow I feel like you didn't know that here in this Project we do not use html file to create the login page as this project has built with the Next.js framework. So if possible please make the PR of try to implement your ideas with Next.js. To know further about Next.js you can read it's documentation, it is easily available on the internet. I hope you got what I am trying to say.
> Note: By mistake your previous PR got merged 😜 and then I have to revert the commits but it wasn't counted as it was not correct. Reason I have told you

okay mam np

**Comment by satavshreya** [comment]
@khushishikhu @Aaishpra ready to work on this Anytime :)

**Comment by SnehaVeerakumar** [comment]
I would like to work on this issue.
If assigned, 2 days of time is required to complete this issue

**Comment by Aaishpra** [comment]
Hi @SnehaVeerakumar  thanks for taking interest in this issue, Although I don't need you to complete the work in just 2 days(take as much time as you want) but can you follow the above design for creating the login Page? I will assign you here do ask me if you face any doubts while working here.

**Comment by Aaishpra** [comment]
@satavshreya i will assign you to a different issue

**Comment by SnehaVeerakumar** [comment]
@Aaishpra Yeah sure I will follow the above mentioned design. Thank you for assigning !

**Comment by akshitakanojia** [comment]
@khushishikhu I would like to work on this issue.

**Comment by khushishikhu** [comment]
@akshitakanojia  assigning to you

**Comment by gnikita432** [comment]
@akshitakanojia you still working on this issue?

**Comment by gnikita432** [comment]
@khushishikhu If no one is working on this issue then will you please assign it to me?

**Comment by akshitakanojia** [comment]
yes @gnikita432 I'm working.

**Comment by ambivert-sam** [comment]
> @khushishikhu Mam please review it....I used a bit of animation also
> ![Screenshot (157)](https://user-images.githubusercontent.com/74984661/111902452-8ef8d680-8a63-11eb-9c48-ef60d7063ebe.png)

 @khushishikhu Mam please can you please merge my PR at least for the design I have created for the login page?

**Comment by khushishikhu** [comment]
@ambivert-sam okay make a PR for it. just add your design as a file. I will merge it

**Comment by ambivert-sam** [comment]
Yup, it's merged but you said it will not be counted. Ma'am so the design will be counted then?

**Comment by khushishikhu** [comment]
> Yup, it's merged but you said it will not be counted. Ma'am so the design will be counted then?
> 
> 

Yes it will I have putted the label on that so it will

**Comment by ambivert-sam** [comment]
Thank you ❤

**Comment by akshitakanojia** [comment]
@khushishikhu I'm facing some system issues. Though I've almost completed the work, but due to the system issues I cannot make PR. I request you to assign this issue to someone else. Apologies for the inconvenience.

**Comment by khushishikhu** [comment]
@srishtij2000 yaa sure. Assigning you

**Comment by kumarishalini6** [comment]
@khushishikhu if no one is working on this issue can I work on this??
